product_docs/docs/pgd/5/backup.mdx

```diff
@@ -120,9 +120,9 @@ applied to `N2` until `T2`.
 PostgreSQL PITR is designed around the assumption of changes arriving
 from a single master in COMMIT order. Thus, PITR is possible by 
 scanning through changes until one particular point in time (PIT) is reached.
-With this scheme, you can restore one node to a single point in time
+With this scheme, you can restore one node to a single PIT
 from its viewpoint, for example, `T1`. However, that state doesn't include other
-data from other nodes that  committed near that time but had not yet
+data from other nodes that committed near that time but had not yet
 arrived on the node. As a result, the recovery might be considered to
 be partially inconsistent, or at least consistent for only one
 replication origin.
```

Comment:
In the next sentence, it wasn't clear to me what "this" is. Is it PITR? 

To request PITR, use the standard syntax: 

?

Reply:
Do you mean this sentence?

With this scheme, you can restore one node to a single PIT
from its viewpoint, for example, `T1`.

Reply:
I think... 

"With PostgreSQL POTR, you can..."